qp_klp/Step.py

```diff
@@ -607,6 +613,58 @@ def load_preps_into_qiita(self, qclient):
 
         return df
 
+    def _load_preps_into_qiita2(self, qclient, prep_id, qiita_id, out_dir,
```

Comment:
is `_load_preps_into_qiita()` still being used? If it is not, then suggest adopting that name. If it is, then the naming needs to be more appropriate

Reply:
Thanks for pointing it out! This code is vestigial.